Representable/representable
# Add new draw tool [@k-y-le]

Kyle's select tool PR. 

**changes**

removed mapbox draw tool (user polygon)
implemented hover + click to highlight select tool
change selection area with input control (1-100)
'Toggler Eraser' control un-highlights selected areas
'Clear Selection' control clears the selection
made it so that user_polygon field is no longer required in db, and census_blocks_polygon_array is
other changes to flow, including: wording of units above the map, removing the success message from displaying upon drawing a map
this works for the current six states we have 'available' : mi, va, pa, nj, tx, il

**to test**

go to entry page
try to draw a district, play around with the tools
save entry & check
still to do (future PR)

validate contiguity of polygons (currently, can submit any selection (of any size as well) without issue, i wrote some comments on how to possibly move forward with turf.js#intersect but that's tbd)
error messages + reloading the map on incomplete entry behaves a bit oddly currently
